frontend/src/components/modals/Modals.tsx

```diff
@@ -0,0 +1,32 @@
+import { ModalsWrapper } from "./Modals.styled";
+
```

Comment:
remove this blank spaces in all files between imports

Reply:
ok